.github/actions/spelling/only.txt

```diff
@@ -0,0 +1 @@
+\.md$
```

Comment:
Might as well go ahead and add the newline here so we aren't constantly getting the warning below when this is merged.